py2many/cli.py

```diff
@@ -215,6 +217,11 @@ def _process_once(settings, filename, outdir):
         if subprocess.call([*settings.formatter, output_path]):
             print(f"Error: Could not reformat: {output_path}")
             return False
+    if settings.ext == ".kt":
+        # ktlint formatter needs to be invoked twice before output is lint free
```

Comment:
I hope someone writes a better linter for kotlin. It's one of the slower backends already

Reply:
Meh, we just need to get this fixed.
https://github.com/pinterest/ktlint/issues/1116 seems similar, but not quite the same issue.